### PR TITLE
Dev/csv export 8122

### PR DESCRIPTION
**Pull request by mcantelon**
This PR adds bulk CSV export of descriptions, either RAD or ISAD, and does a little bit of cleanup/refactoring of the bulk export class. A parent CSV export class exists upon which other CSV export child classes can be created.

### CODE REVIEW

File: lib/flatfile/QubitFlatfileExport.class.php [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+      {
+        $this->fileIndex++;
+        unset($this->currentFileHandle);
+      }
+
+      // Generate filename
+      $filename = exportBulkBaseTask::generateSortableFilename($this->fileIndex, 'csv', $this->standard);
+      $filePath = $this->path .'/'. $filename;
+    }
+    else {
```
Comment (MikeFE) [inline lib/flatfile/QubitFlatfileExport.class.php]: minor style issue, braces on their own lines always

File: lib/flatfile/csvInformationObjectExport.class.php [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+  protected function setAlternativeIdentifierColumns()
+  {
+    $properties = $this->resource->getProperties(null, 'alternativeIdentifiers');
+
+    $alternativeIdentifiers      = array();
+    $alternativeIdentifierLabels = array();
+
+    foreach($properties as $property)
+    {
+      array_push($alternativeIdentifierLabels, $property->name);
```
Comment (MikeFE) [inline lib/flatfile/csvInformationObjectExport.class.php]: http://php.net/manual/en/function.array-push.php seems to indicate in the note if you're just pushing a single value to an array, $ary[] = 123 is slightly better than array_push(). But I don't think it's a big deal.

File: lib/job/arBaseJob.class.php [hunk 3]
```diff
@@ -135,7 +135,7 @@ private function formatLogMsg($message)
   /**
    * Clear various Qubit classes' caches.
    */
-  private function clearCache()
+  static function clearCache()
```
Comment (MikeFE) [inline lib/job/arBaseJob.class.php]: I think this method makes more sense at this point put into lib/helper/QubitHelper.php as something named clear_all_orm_cache() or similar. It seems kind of weird to be in arBaseJob if it's being used way elsewhere in the code base.

Comment (mcantelon) [inline lib/job/arBaseJob.class.php]: Agreed. I'll look into making it a helper.

**Pull request closed without merging**